### PR TITLE
New Abductor Uniforms

### PR DESCRIPTION
**Pull request by Metroidlover**
:cl: MetroidLover
add: New Abductor uniforms
/:cl:

Saw these over on Oracle before it's plug got pulled, and I thought they looked nice and would be better than grey jumpsuits

Art by Quantum-M
forward
left
right
up

Let me know if you want any more pictures of different equipment,background, or pose.

### CODE REVIEW

**Comment by Metroidlover** [comment]
Wait wut, I never touched a map file

**Comment by subject217** [comment]
The icon dmis are causing the problem, not mapdiff/appveyor.

**Comment by Bawhoppen** [comment]
Looks like you've deleted the new hazard jumpsuit. (Also I'm not sure the directional on-mobs go in that file...)

**Comment by Steelpoint** [comment]
Can you show us how they look in game, by themselves and with the standard equipment Abductors wear?

File: code/modules/antagonists/abductor/equipment/abduction_gear.dm [hunk 1]
```diff
@@ -794,3 +794,12 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 	airlock_type = /obj/machinery/door/airlock/abductor
 	material_type = /obj/item/stack/sheet/mineral/abductor
 	noglass = TRUE
+
+/obj/item/clothing/under/abductor
+	desc = "The most advanced form of jumpsuit known to reality, looks uncomfortable."
+	name = "alien jumpsuit"
+	icon = 'icons/obj/abductor.dmi'
```
Comment (subject217) [inline code/modules/antagonists/abductor/equipment/abduction_gear.dm]: This line needs to be removed, and you need to put the obj icon in /icons/obj/clothing/uniforms.dmi

The clothing icons themselves (not the obj icon) go in /icons/mob/under.dmi

The other file you added in abductor.dmi appears to serve no purpose and should be removed.

Also you need to update your branch as it appears it didn't have the hazard suit obj icon, hence why the icon diff bot shows you removing it.

Comment (subject217) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: Also, you should post some pics or a short video of how it looks ingame on the actual lads.

Comment (Metroidlover) [reply code/modules/antagonists/abductor/equipment/abduction_gear.dm]: Thanks for the feedback, I'm still very new to this, I'll start getting this done.

**Comment by Metroidlover** [comment]
Wait, did I mess with screw up something ELSE?

**Comment by fluffe9911** [comment]
uh look at the files changed section

**Comment by Metroidlover** [comment]
Huh, well somehow a bunch of files are having the code removed and readded for no discernable reason.

**Comment by subject217** [comment]
uh oh

**Comment by Metroidlover** [comment]
Was sorta leaning towards that solution at this point anyway, see ya in the next PR

**Comment by SpookyWitch** [comment]
Now these look stylish. I like them.

**Pull request closed without merging**